### DISCUSSION TITLE
Tailwind completely disappears in Next.js build step

### DISCUSSION THREAD

**Discussion by nico-bachner** [tailwindlabs/tailwindcss#3244] [Help] [answered]
All of Tailwind's styles disappear when running Next.js' `next build`/ `npm run build` command.
Everything works perfectly in development.
What could be the cause of this?

Using:
- Next.js v10.0.4
- Tailwind v2.0.2
- Postcss v8.2.2

**Comment by nico-bachner** [comment, marked as answer]
Problem solved 🎉 

`purge` option in tailwind config was set to `components/**/*.js` and my components are `jsx`
changing to `components/**/*.jsx` solved the issue.

**Reply by cloudders-cloud** [reply]
Thank you for that answer here. I had issues with `dev` where I haven't added my components to the `content` directive. After adding all my component extensions there everything worked.
